MediaTracker/app/src/main/res/layout/activity_search_result_details.xml

```diff
@@ -0,0 +1,244 @@
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:id="@+id/viewOne"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_weight="60"
+                android:orientation="horizontal" />
+
+            <LinearLayout
+                android:id="@+id/viewTwo"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_weight="40"
+                android:orientation="horizontal" />
+
+        </LinearLayout>
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:paddingBottom="@dimen/activity_vertical_margin"
+            android:paddingLeft="@dimen/activity_horizontal_margin"
+            android:paddingRight="@dimen/activity_horizontal_margin"
+            android:paddingTop="@dimen/activity_vertical_margin">
+
+
+            <RelativeLayout
+                android:id="@+id/rl_book_details_wrapper"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content">
+
+                <FrameLayout
+                    android:id="@+id/fl_image_details_header"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content">
+
+                    <ProgressBar
+                        android:id="@+id/pb_rss_item"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:indeterminate="true"
+                        android:indeterminateOnly="true" />
+
+                    <ImageView
+                        android:id="@+id/iv_img_details_placeholder"
+                        android:layout_width="75dp"
+                        android:layout_height="108dp"
+                        android:layout_alignParentLeft="true"
+                        android:layout_alignParentTop="true"
+                        android:layout_alignTop="@+id/tv_book_details_title"
+                        android:layout_marginRight="15dp"
+                        android:scaleType="centerCrop"
+                        android:src="@drawable/ic_book_black" />
+
+                </FrameLayout>
+
+                <TextView
+                    android:id="@+id/tv_book_details_title"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentTop="true"
+                    android:layout_marginTop="0dp"
+                    android:layout_toRightOf="@id/fl_image_details_header"
+                    android:paddingTop="0dp"
+                    android:text="The Caves of Steel (Robot Series #1) "
```

Comment:
Do we want this to be in? If you won't `setText()` on this `TextView` , the text that you've set here will appear by default, is this intended behaviour?